common/src/test/java/tachyon/master/block/BlockIdTests.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.master.block;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public final class BlockIdTests {
+
+  @Test
+  public void testCreateBlockIdWithMaxSequenceNumber() {
+    Assert.assertEquals(33554431L, BlockId.createBlockId(1, BlockId.getMaxSequenceNumber()));
+    Assert.assertEquals(16777215L, BlockId.createBlockId(0, BlockId.getMaxSequenceNumber()));
+    Assert.assertEquals(4294967295L, BlockId.createBlockId(255, BlockId.getMaxSequenceNumber()));
+  }
+
+  @Test
+  public void testCreateBlockId() {
+    Assert.assertEquals(16797216L, BlockId.createBlockId(1, 20000L));
+    Assert.assertEquals(20000L, BlockId.createBlockId(0, 20000L));
+  }
+
+  @Test
+  public void testGetContainerIdAndSequenceNumber() {
+    Assert.assertEquals(1L, BlockId.getContainerId(33554431L));
+    Assert.assertEquals(16777215, BlockId.getSequenceNumber(33554431L));
+    Assert.assertEquals(255L, BlockId.getContainerId(4294967295L));
+    Assert.assertEquals(16777215, BlockId.getSequenceNumber(4294967295L));
```

Comment:
The max sequence number is defined as a private field in BlockId. Maybe we can just make it public and remove this test.